java/src/jmri/jmrix/loconet/LnPortController.java

```diff
@@ -50,7 +50,7 @@ public boolean okToSend() {
 
     protected boolean mTurnoutNoRetry = false;
     protected boolean mTurnoutExtraSpace = false;
-
+    protected boolean mInterrogate;
```

Comment:
Might be good to put in an explicit initializer `= false` to make that clear.

Reply:
Done

Reply:
> Done

Is this done? The code is still:
`protected boolean mInterrogateAtStart;`
and not
`protected boolean mInterrogateAtStart = false;`
or
`protected boolean mInterrogateAtStart = true;`

Reply:
Apparently I did the rename before the fix.